<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menix</class>
 <widget class="QMainWindow" name="menix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menix v0.2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_prueba">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>mensaje de prueba</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Probar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdCola">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="numDigits">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
